src/symbolize/gimli.rs

```diff
@@ -119,10 +118,14 @@ impl<'data> Context<'data> {
         dwp: Option<Object<'data>>,
     ) -> Option<Context<'data>> {
         let mut sections = gimli::Dwarf::load(|id| -> Result<_, ()> {
-            if cfg!(not(target_os = "aix")) {
+            #[cfg(not(target_os = "aix"))]
+            {
                 let data = object.section(stash, id.name()).unwrap_or(&[]);
                 Ok(EndianSlice::new(data, Endian))
-            } else if let Some(name) = id.xcoff_name() {
+            }
+
+            #[cfg(target_os = "aix")]
+            if let Some(name) = id.xcoff_name() {
```

Comment:
Mm, this means normal builds don't test if this part of the AIX path works tho'...

Reply:
reverted